Messaging.md

```diff
@@ -379,15 +395,21 @@ See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 This type contains an `oracle_event` and a signature certifying its origination.
 See [the Oracle specifications](./Oracle.md#oracle-announcements) for more details.
 
-#### Version 0 `oracle_announcement`
+#### Version 1 `oracle_announcement`
 
-1. type: 55332 (`oracle_announcement`)
+1. type: 55354 (`oracle_announcement`)
 2. data:
-   * [`signature`:`annoucement_signature`]
-   * [`x_point`:`oracle_public_key`]
+   * [`signature`:`annoucement_pok_signature`]
```

Comment:
I guess pok means proof of knowledge? Unless I missed it it'd be nice to write it down.